tutorials/baseline/auto_trainer_example.ipynb

```diff
@@ -0,0 +1,571 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "obvious-chicago",
+   "metadata": {},
+   "source": [
+    "# Auto Trainer Tutorial\n",
+    "\n",
+    "Here we want to show you an maximally easy example how to use Knodle out-of-the box. Therefor this tutorial contains\n",
+    "the following steps:\n",
+    "1. Download from Knodle Server and load into memory\n",
+    "2. Initialize model and data for a TF_IDF and Logistic Regression example\n",
+    "3. Use AutoTrainer to use 2 weak training strategies\n",
```

Comment:
.. but now in the tutorial, there is only a majority vote example, am I wrong? 
I think the idea of showing 2 weak training strategies is good, e.g.:
```
maj_trainer = AutoTrainer(
    name="majority", "knn", "dscrossweigh"
...
)
```

Reply:
Not sure if we discussed it's part of my PR